holoviews/core/util.py

```diff
@@ -710,3 +710,47 @@ def get_param_values(data):
 def wrap_tuple(unwrapped):
     """ Wraps any non-tuple types in a tuple """
     return (unwrapped if isinstance(unwrapped, tuple) else (unwrapped,))
+
+def unpack_group(group, getter):
+    for k, v in group.iterrows():
+        obj = v.values[0]
+        key = getter(k)
+        if hasattr(obj, 'kdims'):
+            yield (key, obj)
+        else:
+            yield [((), (v.ix[0],))]
+
+
+def ndmapping_groupby_pandas(ndmapping, dimensions, container_type, group_type, **kwargs):
```

Comment:
From what I can see, this is actually used at the ``MultiDimensionalMapping`` level. Just a minor comment, as this name is certainly less unwieldy than ``multidimensional_groupby_pandas`` and I doubt this would cause much confusion.